siliconcompiler/tools/vpr/vpr.py

```diff
@@ -26,6 +26,48 @@ def make_docs(chip):
     return chip
 
 
+def assemble_options(chip, tool):
+
+    step = chip.get('arg', 'step')
+    index = chip.get('arg', 'index')
+    # task = chip._get_task(step, index)
+
+    options = []
+
+    topmodule = chip.top()
+    blif = f"inputs/{topmodule}.blif"
+
+    for arch in chip.get('fpga', 'arch'):
+        options.append(arch)
+
+    options.append(blif)
+
+    if 'sdc' in chip.getkeys('input'):
+        options.append(f"--sdc_file {chip.get('input', 'fpga', 'sdc', step=step, index=index)}")
```

Comment:
always false. you need to use find_files for this but you cannot do this in `setup()` it must only be called in `pre_process`